q2_boots/core_metrics.py

```diff
@@ -32,24 +33,71 @@ def core_metrics(ctx, table, sampling_depth, metadata,
 
     tables = bootstrapped_tables.values()
 
+    alpha_metric_names = alpha_metrics
+
+    alpha_metrics = {'PHYLO': {}, 'NONPHYLO': {}}
+
+    for metric in alpha_metric_names:
+        if metric in ALPHA_METRICS["PHYLO"]["IMPL"] or metric in\
+           ALPHA_METRICS["PHYLO"]["UNIMPL"]:
+            metric = ALPHA_METRICS['NAME_TRANSLATIONS'][metric]
+            alpha_metrics['PHYLO'][metric] = ctx.get_action('diversity_lib',
+                                                            metric)
+        else:
+            if metric in ALPHA_METRICS['NONPHYLO']['IMPL']:
+                metric = ALPHA_METRICS['NAME_TRANSLATIONS'][metric]
+                alpha_metrics['NONPHYLO'][metric] = ctx.get_action('diversity_lib',
+                                                                   metric)
+            else:
```

Comment:
I would probably explicitly check all the cases not have a default case. If more options are added later it will be harder to figure out where/how to support them with this default case here vs an explicit check. There could maybe be a default case that just errors so if someone adds a case to the accepted input then forgets to support it here things explode on them.

Reply:
I don't really know, I would imagine we could in theory have multiple different phylogenetic methods for instance